.github/workflows/deploy_docs.yml

```diff
@@ -1,6 +1,7 @@
 name: publish mkdocs to github pages
 
 on:
+  workflow_dispatch:
   push:
     branches:
       - unidata_18.1.1
```

Comment:
Does this branch need to be updated to unidata_18.2.1?

Reply:
good catch! should be done now